shared/constants/snaps/permissions.ts

```diff
@@ -16,10 +16,10 @@ export const EndowmentPermissions = Object.freeze({
 
 // Methods / permissions in external packages that we are temporarily excluding.
 export const ExcludedSnapPermissions = Object.freeze({
-  eth_accounts:
+  'endowment:caip25':
```

Comment:
Is this an "excluded snap permission"? Versus an "Excluded snap endowment". It is an endowment, and it seems that this is currently being passed into `buildSnapRestrictedMethodSpecifications` and treated as a restricted method permission.

Reply:
Good point. Let me verify if I did this on purpose or not and get back to you